rip_docker.sh

```diff
@@ -13,28 +13,53 @@ if [ "$#" -ne 2 ]; then
     exit -1
 fi
 
-# Flags determined by examining docker-wine with:
-# --as-me
-# --workdir
-# --xvfb
-# --name 
-# https://github.com/scottyhardy/docker-wine/blob/master/docker-wine
-docker run \
-       -it \
-       --rm \
-       --volume=${2}:/data \
-       --env=USER_NAME=${USER} \
-       --env=USER_UID=$(id -u ${USER}) \
-       --env=USER_GID=$(id -g ${USER}) \
-       --env=USER_HOME=${HOME} \
-       --workdir=/home/${USER} \
-       --env=USE_XVFB=yes \
-       --env=XVFB_SERVER=:95 \
-       --env=XVFB_SCREEN=0 \
-       --env=XVFB_RESOLUTION=320x240x8 \
-       --env=DISPLAY=:95 \
-       --hostname=bruker-ripper \
-       --name=bruker-ripper \
-       --shm-size=1g \
-       --env=TZ=America/Los_Angeles \
-       ${1}
+## Support running in parallel:
```

Comment:
I've been running this script on my local computer to avoid filesystem slowness, so this is nice for parallelization, but perhaps better to be in new script..?